languages/java8/template/src/main/java/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,26 @@
+package algorithmia;
+import com.algorithmia.algorithmHandler.*;
+import com.algorithmia.*;
+import java.util.HashMap;
+
+public class Algorithm {
+    String Apply(String input, HashMap<String, String> context) throws Exception{
```

Comment:
I'm not sure what the prevailing opinion is, but I find myself wanting the template to use json input/output (not merely a string) which most algos end up needing, more than trying to showcase loading (although, maybe we need both?)

Reply:
Agreed, I'd like the advanced variant to do both.